pkg/option/config.go

```diff
@@ -2411,3 +2453,23 @@ func getDefaultMonitorQueueSize(numCPU int) int {
 	}
 	return monitorQueueSize
 }
+
+// EndpointStatusValues returns all available EndpointStatus option values
+func EndpointStatusValues() []string {
+	return []string{
+		EndpointStatusPolicy,
+		EndpointStatusHealth,
+		EndpointStatusControllers,
+		EndpointStatusLog,
+		EndpointStatusState,
+	}
```

Comment:
Does not hurt to return this by order with `sort.Strings()` or ordering them manually. before returning this slice.